src/components/about/About.component.tsx

```diff
@@ -1,33 +1,40 @@
+import { clsx } from 'clsx';
+
 import styles from './about.module.css';
 
-function App() {
+const About = ({ theme }: { theme: string }) => {
     const name = 'Lesia';
-
     return (
-        <main className={styles.main}>
+        <main
+            className={clsx({
+                [styles.main]: true,
+                light_theme: theme === 'light',
+                dark_theme: theme === 'dark',
+            })}
```

Comment:
`clsx(styles.main, {
    light_theme: theme === 'light',
    dark_theme: theme === 'dark',
})`

can we use like this?

Reply:
fixed